clinica/pipelines/statistics_surface/_outputs.py

```diff
@@ -0,0 +1,226 @@
+"""This file contains functions to write results of the
+StatisticsSurface Pipeline to disk.
+"""
+
+import warnings
+from os import PathLike
+from typing import Dict, List, Union
+
+import numpy as np
+
+
+def _save_results_to_json(results: Dict, filename_root: PathLike, verbose: bool = True):
+    """Write the provided results to JSON format.
+
+    Parameters
+    ----------
+    results : Results to write to disk.
+    filename_root : Filename root common to all output files.
+    verbose : Verbose mode.
+    """
+    import json
+
+    out_json_file = str(filename_root) + "_results.json"
+    if verbose:
+        print(f"--> Writing results to JSON in {out_json_file}...")
```

Comment:
Let's have all the print messages use `cprint` to be able to channel through the logger. That would automatically handle the verbose flag.

Reply:
Thanks for the suggestion @omar-rifai ! 👍 
Implemented in 10b4c9c